trigger/patch.go

```diff
@@ -203,9 +203,16 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 		PastTenseStatus:   t.data.Status,
 		apiModel:          &api,
 		githubState:       message.GithubStatePending,
-		githubContext:     "evergreen",
 		githubDescription: "tasks are running",
 	}
+
+	if t.patch.IsChild() {
+		GitHashLastFour := t.patch.Githash[len(t.patch.Githash)-4:]
+		data.githubContext = fmt.Sprintf("evergreen/%s/%s", t.patch.GithubPatchData.BaseBranch, GitHashLastFour)
```

Comment:
I don't think the child patch has `GithubPatchData.BaseBranch` populated, though it wouldn't be hard to populate.

Reply:
good point, I'll look into setting it to the project.